src/cpp/session/SessionOptions.cpp

```diff
@@ -274,7 +274,7 @@ core::ProgramStatus Options::read(int argc, char * const argv[])
          value<bool>(&autoReloadSource_)->default_value(false),
          "Reload R source if it changes during the session")
       ("r-compatible-graphics-engine-version",
-         value<int>(&rCompatibleGraphicsEngineVersion_)->default_value(11),
+         value<int>(&rCompatibleGraphicsEngineVersion_)->default_value(12),
```

Comment:
That is probably just forwarding from one place (the session config file) with some defaults that we never updated. I'd set it to 12 everywhere.